claims.go

```diff
@@ -63,6 +64,11 @@ func getClaim(logger log.Logger, allClaims map[string]interface{}, claim string)
 	//
 	// Or we fork and/or PR go-oidc.
```

Comment:
We should modify this comment to remove the suggested approach since we're going to use it. I do think it's worth leaving part of this comment in place to explain the non-obvious bits around json.Number.